app/controllers/openid_token_proxy/callback_controller.rb

```diff
@@ -0,0 +1,20 @@
+module OpenIDTokenProxy
+  class CallbackController < ApplicationController
+    def handle
+      unless code = params[:code]
+        render nothing: true, status: :bad_request
```

Comment:
That's the thing, this page will be visible in the browser (briefly) before redirecting to whatever application or web app *actually* needed the token. But in practice I haven't seen it fail all that often.